spec/lib/cap/authors_poller_spec.rb

```diff
@@ -128,14 +130,14 @@
       it 'harvests for new authors marked harvestable' do
         expect(author).to receive(:harvestable?).and_return(true)
         subject.process_record(author_record)
-        queue = subject.instance_variable_get('@new_or_changed_authors_to_harvest_queue')
+        queue = subject.instance_variable_get('@new_authors_to_harvest_queue')
```

Comment:
This test was only ever working before because we weren't differentiating between new or changed authors when building the queue.   Previously the queue was changing because because it was considered a changed author, not a new author.